cli/src/commands/list.ts

```diff
@@ -8,7 +8,7 @@ export const listFilesCommand = new Command('list')
   .description('Lists files from the given website on Massa blockchain')
   .option('-a, --address <address>', 'Address of the website to list')
   .action(async (options, command) => {
-    const globalOptions = command.parent?.opts()
+    const globalOptions = command.optsWithGlobals()
 
     if (!globalOptions) {
       throw new Error(
```

Comment:
`command.optsWithGlobals()` returns an `OptionsValue`, so the `if (!globalOptions)` will never be triggered. We can remove it.

Sorry, I couldn't do a suggestion because Github does not allow me to select code that wasn't modified, nor suggest changes on deleted lines.